packages/core/bootstrap/src/lib/cache/index.ts

```diff
@@ -141,9 +156,10 @@ export const withCache: Middleware = async (execute, options = defaultOptions())
     const coalescingKey = _getCoalescingKey(key)
     const endMetrics = metrics.beginObserveCacheExecutionDuration({
       participantId: key,
-      feedId: data.debug?.feedId,
+      feedId: data.debug?.feedId || 'unknown',
     })
     let maxAge = _getRequestMaxAge(data) || _getDefaultMaxAge(data)
+    if (maxAge < options.minimumAge) maxAge = options.minimumAge
```

Comment:
This causes the `if (maxAge < 0) {` check in `_cacheOnSuccess` to never execute, is this what we want? I know that when data is being from from WS to cache, the maxAge is set to `-1`, we may want to check that interaction with this code change

Reply:
Moved inside `_cacheOnSuccess`